db/db.go

```diff
@@ -0,0 +1,33 @@
+package db
+
+import (
+	"database/sql"
+	"os"
+
+	_ "modernc.org/sqlite"
+)
+
+var DB *sql.DB
+
+func Connect(databaseFile string) error {
+	var err error
+	DB, err = sql.Open("sqlite", databaseFile)
+	if err != nil {
+		return err
+	}
+
+	if err = DB.Ping(); err != nil {
+		return err
```

Comment:
Лучше все ошибки оборачивать, чтобы быстрее искать где они возникли
```suggestion
		return fmt.Errorf("error while making ping to db: %w", err)
```